kernel/kernel-impl/src/test/java/org/sakaiproject/content/impl/serialize/impl/test/MockTimeService.java

```diff
@@ -21,253 +21,22 @@
 
 package org.sakaiproject.content.impl.serialize.impl.test;
 
-import java.time.Instant;
-import java.time.LocalDate;
-import java.time.format.FormatStyle;
-import java.util.Date;
-import java.util.GregorianCalendar;
-import java.util.Locale;
-import java.util.TimeZone;
-
-import org.sakaiproject.time.api.Time;
-import org.sakaiproject.time.api.TimeBreakdown;
-import org.sakaiproject.time.api.TimeRange;
 import org.sakaiproject.time.api.TimeService;
+import static org.mockito.Mockito.*;
```

Comment:
Please don't use * imports unless you're actually utilizing all classes within the package. Otherwise, import only the classes/packages needed in this file.